packages/clerk-js/src/ui/customizables/parseAppearance.ts

```diff
@@ -44,6 +44,9 @@ const defaultLayout: ParsedLayout = {
   shimmer: true,
   animations: true,
   unsafe_disableDevelopmentModeWarnings: false,
+  captchaTheme: 'light',
+  captchaSize: 'normal',
```

Comment:
Does this height value need to change based on the `captchaSize`? https://github.com/clerk/javascript/blob/0f95982551565f11032b4d018d89746a081c1b76/packages/clerk-js/src/utils/captcha/turnstile.ts#L188